README.md

```diff
@@ -1,36 +1,100 @@
+
+Note: `cadence-web` can be connected to multiple clusters by adding comma-delimted entries for `CADENCE_GRPC_PEERS`, `CADENCE_GRPC_SERVICES_NAMES` & `CADENCE_CLUSTERS_NAMES` for each cluster (each cluster values are grouped by it's index within the Comma-delmited lists).
+
+
+### Using cadence-web
+
+Latest version of `cadence-web` is included in `cadence` composed docker containers that can be found [here][cadence]. To start using it clone `cadence` repo and run the following command
+```
+docker-compose -f docker/docker-compose.yml up
+```
+
+### Building & developing cadence-web 
+
+`cadence-web` requires node `v18` or greater to be able to run correctly.
 
-```bash
+#### Creating a production build
+
+To create a production build, follow this steps:
+
+1. Install npm packages and download idls
+```
+npm install && npm run install-idl && npm run generate:idl
+```
+2. Build the project files
+```
+npm run build
+```
+3. After building the code, the server can be started by running this command from the same directory that includes the build
+```
+npm start
+```
+4. The webapp can be accessed now through `localhost:8088` (port can be changed using `CADENCE_WEB_PORT` environment variable)
+
+#### Running development environment
+
+To create a production build, follow this steps:
+
+1. Install npm packages and download idls
+```
+npm install && npm run install-idl && npm run generate:idl
+```
+2. Build the project files
+```
 npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+Note: For contribution we recommend using dev containers check `VSCode Dev Containers` section for more information
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+#### Using VSCode Dev Containers
 
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+1. Set up the [Remote Containers plugin](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) in VSCode.
+2. Open the cadence-web directory in VSCode.
+3. Use the Command Palette to select the 'Reopen folder in Container' option. `npm install` should run automatically in the container upon doing so.
+4. Run the app in the container with `npm run dev`.
+5. Open `localhost:8088` (or the custom Cadence Web port you have defined) to load the webapp.
 
-## Learn More
+#### NPM scripts
 
-To learn more about Next.js, take a look at the following resources:
+`build`: `Generates production builds`
+`start`: `Starts a production build`
+`dev`: `Run development server`
+`install-idl`: `Downloads idl files required for building/running the project`
+`generate:idl`: `Move idl files inside the project and generate typescript types for them`
+`test`: `Run all test cases`
+`test:unit`: `Run all unit tests`
```

Comment:
You can mention that this does not support the `-u` flag, and tell users they need to use browser/node for this